common/src/main/java/tds/common/web/interceptors/RestTemplateEventLoggingInterceptor.java

```diff
@@ -0,0 +1,58 @@
+package tds.common.web.interceptors;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.HttpRequest;
+import org.springframework.http.client.ClientHttpRequestExecution;
+import org.springframework.http.client.ClientHttpRequestInterceptor;
+import org.springframework.http.client.ClientHttpResponse;
+
+import java.io.IOException;
+
+import tds.common.logging.EventLogger;
+
+import static tds.common.logging.EventLogger.Checkpoint.SERVICE_CALL;
+import static tds.common.logging.EventLogger.Checkpoint.SERVICE_RETURN;
+import static tds.common.logging.EventLogger.EventData.HTTP_HEADERS;
+import static tds.common.logging.EventLogger.EventData.RESPONSE_CODE;
+import static tds.common.logging.EventLogger.EventData.RESPONSE_CODE_TEXT;
+
+/**
+ * An interceptor for logging REST requests and responses
```

Comment:
If we need two of these (per @wtritch's comment above) can we enhance the JavaDoc slightly to call out that this is for event logging whereas the other one is for standard logging?  We now have two similarly named classes, and I think the JavaDoc should call out the difference.